src/middlewared/middlewared/plugins/support.py

```diff
@@ -184,10 +179,10 @@ async def new_ticket(self, job, data):
         """
         Creates a new ticket for support.
         This is done using the support proxy API.
-        For FreeNAS it will be created on Redmine and for TrueNAS on SupportSuite.
+        For FreeNAS it will be created on JIRA and for TrueNAS on Salesforce.
```

Comment:
Still a reference to `FreeNAS`. Sentence should probably be `For TrueNAS Core/Scale it will be created on JIRA, and for TrueNAS Core/Scale Enterprise on SalesForce.`